.github/workflows/release-docs.yml

```diff
@@ -0,0 +1,63 @@
+name: Release Documentation
```

Comment:
I would really prefer if this happened in the already existing release pipeline or at least is triggered from there and not standalone.
I want to prevent a situation where we have a whole cascade of pipelines going off when we do a release. I'm concerned with losing oversight with many different things happening at different places.

Reply:
changed to a workflow call